src/Microsoft.TestPlatform.CoreUtilities/FeatureFlag/FeatureFlag.cs

```diff
@@ -57,6 +57,9 @@ internal partial class FeatureFlag : IFeatureFlag
     // Disable the SerialTestRunDecorator
     public const string DISABLE_SERIALTESTRUN_DECORATOR = VSTEST_ + nameof(DISABLE_SERIALTESTRUN_DECORATOR);
 
+    // Disable setting UTF8 encoding in console.
+    public const string DISABlE_UTF8_CONSOLE_ENCODING = VSTEST_ + nameof(DISABlE_UTF8_CONSOLE_ENCODING);
```

Comment:
Any reason DISABLE has a lowercase 'l' ?

Reply:
I wish I could say I am testing your attention, but I have no idea why. My fingers must have typed it that way. I sometimes type 0 instead of o or O, and vice versa. I guess this is another quirk of l L and 1 looking the same and my brain working in visual way.